addon/components/bs-form.js

```diff
@@ -426,7 +423,10 @@ export default class Form extends Component {
     let model = this.model;
 
     this.incrementProperty('pendingSubmissions');
-    this.onBefore(model);
+
+    if (typeof this.onBefore === 'function') {
+      this.onBefore(model);
+    }
```

Comment:
I think this is also the path forward for glimmer component refactoring. After refactoring to glimmer components it would look like this:

```js
    if (typeof this.args.onBefore === 'function') {
      this.args.onBefore(model);
    }
```